ComunityMembers/RepositoryTemplate/AddressRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿using CommunityModels.MembershipModels.Details;
+using CommunityRepository.DatabaseContext;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommunityRepository
+{
+    public class AddressRepository : AbstractRepository<Address>
```

Comment:
Please add description to all public members and classes and constructors, here and everywhere